### PR TITLE
Session proposal

### PR DESCRIPTION
**Pull request by Manuss20**
Session proposal

### CODE REVIEW

**Comment by manuparra** [comment]
Gracias @Manuss20 !

File: talks/manuelsanchez.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Manuel Sánchez Rodríguez
+
+## Presenter
+
+Manuel Sánchez Rodríguez
+
+## Abstract (200 words max.) 
+
+This session is designed for developers who want to learn how to create plugins using Semantic Kernel and Azure OpenAI. You will learn how to create plugins that can be used across all of the major AI apps and services like ChatGPT, Bing, and Microsoft 365. This will allow you to increase the reach of your AI capabilities without rewriting your code.
```
Comment (JJ) [inline talks/manuelsanchez.md:13]: Thanks, Manuel, but Databeers is not really suited for this kind of session. We're looking for fast (as in 10 minutes) talk that highlight some aspect of data manipulation or visualization or machine learning.

**Review by JJ — COMMENTED** [review]
You will probably want to submit this session to the general track; to be honest, I don't know if it's open yet, but check out [the LinkedIn profile](https://www.linkedin.com/posts/agenciadigitalandalucia_savethedate-andalucia23-ia-activity-7113534266554105856-JrkS?utm_source=share&utm_medium=member_desktop), it will probably open the call for proposals soon.

File: talks/manuelsanchez.md (hunk 1 above)
Comment (Rms78) [reply talks/manuelsanchez.md:13]: We could check the available slots in the workshop space, in case you're interested.

**Pull request closed without merging**